src/PrestaShopBundle/DependencyInjection/Compiler/CoreDoctrineCompilerPass.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Two things : the copyright date & `declare(strict_types=1);`

Reply:
It's for the 1.7.6.x we can't use the strict types